ansible/roles/test/tasks/fdb.yml

```diff
@@ -61,3 +61,7 @@
       - vlan_ip='{{minigraph_vlan_interfaces[0]['addr']}}'
     ptf_extra_options: "--relax --debug info --log-file /tmp/fdb_test.FdbTest.{{lookup('pipe','date +%Y-%m-%d-%H:%M:%S')}}.log "
   
+- block:
+    - name: Include fdb_mac_expire.yml
+      include: fdb_mac_expire.yml
+  when: "run_fdb_mac_expire|bool == True"
```

Comment:
can you put as a separate test? fdb_mac_expire.yml?

Reply:
Okie if separate test suite is prefered. I will add below in file ansible/roles/test/vars/testcases.yml.

    fdb_mac_expire:
      filename: fdb_mac_expire.yml
      topologies: [t0, t0-16, t0-64, t0-64-32, t0-116]
      required_vars:
          ptf_host:
          testbed_type: